src/pages/material/material-page/material-page.js

```diff
@@ -27,6 +28,8 @@ const tableTitles = config.tableHeadRowTitles.materials;
 
 const MaterialPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed